api/desecapi/pdns.py

```diff
@@ -98,6 +98,15 @@ def get_zone(domain):
     return r.json()
 
 
+def get_axfr(domain):
+    """
+    Retrieves a dict representation of the zone from pdns
+    """
+    r = _pdns_get(NSLORD, '/zones/' + pdns_id(domain.name) + '/export')
+
+    return r.json()['zone']
```

Comment:
The response body is the zone file, there's no `zone` field and no JSON

Reply:
I read that in the docs, but apparently it's a JSON anyway. Maybe we need to adjust the `Accept` header...

Reply:
I tried it out in my nsmaster container and did not see any JSON (unless I completely misread the output). Anyway, let's double-check

Reply:
Note sure what's going on but this concern is covered by the e2e2 tests.

Reply:
Indeed, we're sending an `Accept: application/json` header, and when that is the case the response is as you observed. When sending '*/*' or no such header, the response is just the plain zonefile, which saves the conversion in both directions.

I've added two commits that (1) change the pdns request interface to allow specifying the Accept: header, (2) makes use of that and directly returns the pdns output as text/dns resource to the client, without any serialization. What do you think about that?

We may consider prepending a line or two, as follows:
```
; Zonefile $domain.name
; Exported from desec.io at $time
```

Reply:
done & done